ci/scripts/start-bosh.sh

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env bash
```

Comment:
Do we need our own start-bosh file or would the one from the docker image `bosh/docker-cpi:main` also work?